Makefile

```diff
@@ -429,9 +429,9 @@ proto-gen:
 
 
 # TODO: Rethink API docs generation
```

Comment:
After it's done I think we can un comment this part.

Reply:
No, I uncomment the code to use it. No idea why the og dev commented it.